content/02.main-text.md

```diff
@@ -264,6 +266,10 @@ The greatest success to date of open collaborative writing is arguably Wikipedia
 Wikipedia scaled encyclopedias [far beyond](https://en.wikipedia.org/wiki/Wikipedia:Size_comparisons) any privately-written alternative.
 These examples illustrate how open collaborative writing can scale scholarly manuscripts where diverse opinion and expertise are paramount beyond what would otherwise be possible.
 
+Papers with hundreds or thousands of authors are on the rise, such as the article describing the experiments and data that led to the discovery of the Higgs Boson [@doi:10.1103/PhysRevLett.114.191803] (5000 authors) and the report of the Drosophila genome [@doi:10.1534/g3.114.015966] (1000 authors).
+Yet _the number of people that participated in writing_ those papers, as opposed to generating and analyzing the data, is not always clear and likely to be far below the number of authors [@doi:10.1038/521263f; @doi:10.1038/nature.2015.17567].
+Manubot provides the scientists involved in large collaborations the opportunity to actively participate, through a public forum, in the writing process.
+
```

Comment:
This paragraph could also fit nicely at the end of the authorship section?

Reply:
I agree, it works well there too.  Should I move it there?

Reply:
I'm leaning towards moving it to the authorship section. One issue with the current manuscript is that ideas are rather scattered. Therefore, I'm generally in favor of putting topics together rather than segregating into methods/discussion.

Also consider whether the phrase: `and helps prevent ghostwriting [@doi:10.1371/journal.pmed.1000023]` earlier in the sentence belongs here.

Reply:
Preventing ghostwriting is an interesting point! Maybe it deserves its own sentence.

Reply:
I moved the mega paper paragraph in 3d4e3c4 and ghostwriting in 9ec30a8.